src/components/ui/AspectRatio/AspectRatio.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef, Children } from 'react'
+import type { HTMLAttributes } from 'react'
+
+import './aspect-ratio.scss'
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /*
+   * The aspect ratio value: Common values are:
+   *
+   * `21:9`, `16:10`, `16:9`, `4:3`, `2:1`
+   */
+  ratio?: string
```

Comment:
I was wondering if we can use these values as `type Ratio = "21:9" | "16:10" | "16:9" | "4:3" | "2:1" [...]`. But I agree that string keep it more flexible :)

Reply:
hmm I think we can add your suggestion right now because we don't need to be flexible here as we are not dealing with custom ratios yet